src/action.js

```diff
@@ -24,6 +25,9 @@ const {
 
 module.exports = async function run() {
   try {
+    const warning = toolkit.getActionRefWarning('fastify/github-action-merge-dependabot')
+    if (warning) core.warning(warning)
```

Comment:
why doesn't the toolkit raise the warning internally so we don't have to use this conditional in user code?